parser.go

```diff
@@ -392,9 +396,10 @@ func GetString(data []byte, keys ...string) (val string, err error) {
 		return string(v), nil
 	}
 
-	s, err := strconv.Unquote(`"` + unsafeBytesToString(v) + `"`)
+	var stackbuf [unescapeStackBufSize]byte // stack-allocated array for allocation-free unescaping of small strings
```

Comment:
Why not? The array is stack allocated, which means a new one is allocated each time the function is called. It should be totally reentrant. If you want an example, see `bytes.Buffer` in the standard library.